### ISSUE TITLE
Autothrottle cuts off RR engines

### ISSUE THREAD

**Issue by edvinasandr** [magknight/787aviatorsTracker#348]
**Describe the bug**
When I turn on Auto Throttle RR engines lose thrust and come back to normal operation only after disengaging a/t

**To Reproduce**
Climb to 20000 ft and try turning on autothrottle with RR engines.

**Expected behavior**
Autothrottle shouldn't kill engines...
**Screenshots**
Video: https://drive.google.com/open?id=1Ya6mHHTof58mROkbNMZvtqfjz3F6rS5r

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - X-Plane version [e.g. 11.41]
 - Version [e.g. 1.5.0B8]

**Additional context**

**Comment by aeroniemi** [comment]
duplicate of #344
